c/src/test_vdm_parse.c

```diff
@@ -17,1864 +17,2029 @@
     This module contains functions to test the vdm_parser.c functions.
     Each function returns a 0 if the test fails and a 1 if it passes.
 
-*/
+ */
 
 
 
 int test_ais2ascii( void )
 {
-    fprintf( stderr, "ais2ascii skipped (passed)\n" );
-    return 1;
+	fprintf( stderr, "ais2ascii skipped (passed)\n" );
```

Comment:
Looks like a bunch of unnecessary indentation changes crept into this file. Please fix and re-push.